iocore/net/P_UDPPacket.h

```diff
@@ -246,6 +246,41 @@ new_incoming_UDPPacket(struct sockaddr *from, char *buf, int len)
 }
 
 TS_INLINE UDPPacket *
+new_incoming_UDPPacket(struct sockaddr *from, IOBufferBlock *block, int len)
+{
+  (void)len;
```

Comment:
Why don't you comment out `len` ?

Reply:
or just remove it like `new_incoming_UDPPacket(struct sockaddr *from, Ptr<IOBufferBlock> &block);`

Reply:
remove it.